File: CS/Form1.cs
using System;
using System.Windows.Forms;

namespace DataInspectorOnInitializeExample
{
    public partial class Form1 : Form {
        public Form1() {
            InitializeComponent();
            dashboardViewer1.Initialized += dashboardViewer1_Initialized;
        }
        async void dashboardViewer1_Initialized(object sender, EventArgs e) {
            await dashboardViewer1.ShowDataInspectorAsync("gridDashboardItem1");
        }
    }
}

File: VB/Form1.vb
Imports System
Imports System.Windows.Forms

Namespace DataInspectorOnInitializeExample
	Partial Public Class Form1
		Inherits Form

		Public Sub New()
			InitializeComponent()
			AddHandler dashboardViewer1.Initialized, AddressOf dashboardViewer1_Initialized
		End Sub
		Private Async Sub dashboardViewer1_Initialized(ByVal sender As Object, ByVal e As EventArgs)
			Await dashboardViewer1.ShowDataInspectorAsync("gridDashboardItem1")
		End Sub
	End Class
End Namespace
